### PR TITLE
Update donate button color

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Updates the color of the donate button to 49-update-donate-button-color to `#D24F2F`

Before:
Screen Shot 2019-09-30 at 5 21 47 PM

After:
Screen Shot 2019-09-30 at 5 21 54 PM

## Why

For #49

## Testing/Questions

Features that this PR affects:

- Donate button styling

### CODE REVIEW

**Comment by kaylima** [comment]
@josh does this also target the sticky nav button?

**Comment by benlk** [comment]
It does not!

File: wp-content/themes/borderzine/less/inc/header.less [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.dontate-btn {
+.donate-btn {
```
Comment (benlk) [inline wp-content/themes/borderzine/less/inc/header.less]: Add `.navbar.sticky-navbar .nav-right #header-extras .donate a span`

Comment (joshdarby) [reply wp-content/themes/borderzine/less/inc/header.less]: Updated in https://github.com/INN/umbrella-borderzine/pull/56/commits/45eb17855976cb2f1bc17706a334d2b96955a704

**Comment by joshdarby** [comment]
> Do we want to do anything with the hover states?

It looks like the hover background color for buttons on borderzine.com currently use the dark gray color. I'll update these to also use it.

**Comment by kaylima** [comment]
@joshdarby we can leave the hover states as-is, thanks!

**Pull request merged**